tox.ini

```diff
@@ -7,6 +7,9 @@
 [tox]
 envlist = py{27,34,35,36}
 
+[flake8]
+ignore = E111, E114
+
```

Comment:
You will need to add flake8 to tox.ini's dependencies entry.

However, I'm not sure why flake8 is appearing here all of a sudden. 
 Doesn't in-toto use Pylint?

Reply:
Oversight on my part. I use flake8 personally and I had added these settings to prevent errors. I was careful not to commit them (till now) :sweat_smile: